index.ts

```diff
@@ -1,11 +1,14 @@
-import * as dotenv from 'dotenv';
+import config from './config';
 import * as mongoose from 'mongoose';
-
 import { start } from './server';
 
-dotenv.config();
-
 mongoose.Promise = global.Promise;
-mongoose.connect('mongodb://localhost/test', { useMongoClient: true });
+
+const { mongoUser, mongoPass } = config;
+const mongoUri = `mongodb://${encodeURIComponent(
```

Comment:
Thoughts on making user and password optional?

Reply:
done